shuttle/templates/schedules/shuttle_schedules.html

```diff
@@ -13,7 +13,20 @@ <h6>Use <a href="https://docs.google.com/spreadsheets/d/1VGuZm58MfKW7TBFG09oGT8e
         <iframe width="100%" height=265 src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTy3uE_5hm1xmXyHdlAUwh3UF31yNw_lE1fACWoj1f3ND-UsMsveEe_FDHLLc8pHxgd4YwmdGBqMWRU/pubhtml?widget=true&amp;headers=false"></iframe>
     </div>
     <div class="center-page">
-         <input class="btn btn-warning btn-sm" type="submit" value="Submit to Database">
+         <input id="database-button" class="btn btn-warning btn-sm" type="submit" value="Submit to Database">
     </div>
 
+    <script>
+    $(document).ready(function() {
+        $("#database-button").click(function () {
+            // Sends google sheets schedule to database (erases old schedule)
+            $.ajax({
+                url: "{{ url_for('SchedulesView:send_schedule_path') }}",
+            }).done(function(data) {
```

Comment:
Add in the .fail() method to catch errors.

https://riptutorial.com/jquery/example/30084/jquery-ajax---success--error--vs---done-----fail--